src/jazayeri_lab_to_nwb/watters/timeseries_interfaces.py

```diff
@@ -0,0 +1,207 @@
+"""Primary classes for timeseries variables.
+
+The classes here handle variables like eye position, reward line, and audio
+stimuli that are not necessarily tied to the trial structure of display updates.
+For trial structured variables, see ../trials_interface.py. For variables
+pertaining to display updates, see ../frames_interface.py.
+"""
+
+import abc
+import json
+import numpy as np
+from pathlib import Path
+from pynwb import NWBFile, TimeSeries
+from pynwb.behavior import SpatialSeries
+from hdmf.backends.hdf5 import H5DataIO
+
+from neuroconv.basetemporalalignmentinterface import BaseTemporalAlignmentInterface
+from neuroconv.utils import DeepDict, FolderPathType, FilePathType
+from neuroconv.tools.nwb_helpers import get_module
+
+
+class TemporalAlignmentMixin(BaseTemporalAlignmentInterface):
```

Comment:
FYI this isn't a mixin any more since the new usage seen in this file is all following linear inheritance

I'd just suggest removing the 'Mixin' from the name to avoid confusion from anyone reading the name of the class

Maybe `TimestampsFromArrayInterface` or something that describes more what it does different from the base class?

Reply:
Yes, thanks!  Done.